src/views/PurchaseOrder.vue

```diff
@@ -151,14 +151,15 @@ export default defineComponent({
         this.mappingName = "";
       },
       getFile(event) {
-        this.file = event.target.files[0];
-        if(this.file){
-          showToast(translate("File uploaded successfully"));
+        const file = event.target.files[0];
+        if(file){
+          this.file = file;
           this.parseFile();
           this.store.dispatch('order/updateFileName', this.file.name);
+          showToast(translate("File uploaded successfully"));
         }
         else {
-          showToast(translate("Something went wrong. Please try again"));
+          showToast(translate("No new file upload. Please try again"));
```

Comment:
Ir works when trying to change the file first time, but if we repeat changing the file more than once, then it does not work.